src/arch/riscv/platform_gen.h.in

```diff
@@ -7,7 +7,9 @@
 #pragma once
 
 #include <config.h>
+
 #define TIMER_CLOCK_HZ @CONFIGURE_TIMER_FREQUENCY@
+
```

Comment:
Two blank lines are added on purpose to improve the template?

Reply:
I've added it to improve readability. Will move this into a separate commit

Reply:
it is ok, just want to confirm it is intended. thanks

Reply:
fixed